VERSION

```diff
@@ -1 +1 @@
-0.0.1
+11.0.2.0
```

Comment:
Is this supposed to be the version of the fed sdk generator in general or specifically to support fed v11?

Reply:
Should be the generator, I reverted it. Thanks for picking it up.